src/test/java/bio/terra/common/ParquetUtils.java

```diff
@@ -21,6 +24,19 @@
 /** Helper test methods for working directly with parquet files */
 public class ParquetUtils {
 
+  public static List<Map<String, Object>> readGcsParquetRecords(
+      GcsPdao gcsPdao, final String url, String googleProjectId) {
+    Configuration config = new Configuration();
+    byte[] contents = gcsPdao.getBlobBytes(url, googleProjectId);
+    try {
+      java.nio.file.Path tempParquetFile = Files.createTempFile("localGcs", ".parquet");
+      Files.write(tempParquetFile, contents);
+      URI uri = tempParquetFile.toUri();
+      return readParquetRecordsFromUri(config, uri);
+    } catch (IOException e) {
+      throw new RuntimeException("Could not write " + url + " to local temp file", e);
```

Comment:
Eeek...we're thinking of downloading parquet files locally?  We should never be doing this in service code. Is this for testing?  If so, we should move this to a testing class so we don't accidentally use it in service code

Reply:
This is in `src/test`.

Reply:
facepalm...I was looking at the package name